EgammaAnalysis/ElectronTools/src/PhotonEnergyCalibratorRun2.cc

```diff
@@ -28,44 +28,84 @@ void PhotonEnergyCalibratorRun2::initPrivateRng(TRandom *rnd)
   rng_ = rnd;
 }
 
-std::vector<float> PhotonEnergyCalibratorRun2::calibrate(reco::Photon &photon, unsigned int runNumber, const EcalRecHitCollection *recHits, edm::StreamID const & id) const
+std::vector<float> PhotonEnergyCalibratorRun2::calibrate(reco::Photon &photon, unsigned int runNumber, 
+							 const EcalRecHitCollection *recHits, edm::StreamID const & id, int eventIsMC) const
 {
   float smear = 0.0, scale = 1.0;
   float aeta = std::abs(photon.superCluster()->eta());
   float et = photon.getCorrectedEnergy(reco::Photon::P4type::regression2) / cosh(aeta);
+
+  if (et < 1.0) {
+    return std::vector<float>(10, photon.getCorrectedEnergy(reco::Photon::P4type::regression2));
+  }
+
   DetId seedDetId = photon.superCluster()->seed()->seed();
   EcalRecHitCollection::const_iterator seedRecHit = recHits->find(seedDetId);
-  unsigned int gainSeedSC = 0;
+  unsigned int gainSeedSC = 12;
```

Comment:
I've made separate PRs for this bug. If you don't merge this  #2 now, please merge the others, otherwise people will find problems.